dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -145,4 +169,34 @@ public Date getLastModified() {
     public void setLastModified(Date lastModified) {
         this.lastModified = lastModified;
     }
+
+    public Map<String, Alias> getAliases() {
+        return aliases;
+    }
+
+    public void setAliases(Map<String, Alias> aliases) {
+        this.aliases = aliases;
+    }
+
+    @ApiModel(value = "WorkflowVersionPathInfo", description = "Object that contains the URL to the workflow and the version tag name.")
+    public static final class WorkflowVersionPathInfo {
+        @ApiModelProperty(value = "URL to workflow.")
```

Comment:
I don't think this is  a URL; it is a path. A url contains a scheme, e.g., http, etc., and I believe the value here is just the Dockstore path, e.g., `github.com/wshands/foo`.

Reply:
changed comment to say 'Dockstore path'